servers/src/main/java/tachyon/master/rawtable/RawTableMasterServiceHandler.java

```diff
@@ -46,6 +55,21 @@ public long createRawTable(String path, int columns, ByteBuffer metadata)
   }
 
   @Override
+  public long createTable(final String path, final int columns, final ByteBuffer metadata, String nonce)
+      throws TachyonTException, ThriftIOException, TException {
+    try {
+      return ((Long) replayCache.get(nonce, new Callable<Object>() {
```

Comment:
Do you think we need to apply this pattern to other non-idempotent operations like mkdir?